### DISCUSSION TITLE
Update cache without deleting it

### DISCUSSION THREAD

**Discussion by smares** [spatie/laravel-responsecache#370] [Q&A]
Is it possible to update the cached version of a page without clearing the cache first? Imagine the following scenario: You have a page that takes 10 seconds to produce an output. The first request takes 10 seconds and will cache the page. Every subsequent request will use the cached version. After 1h you want to update the cache. If you must delete the cache first, every request coming in during that 10-seconds-period it takes for the cache to rebuild will also take 10 seconds. What we would like to do is implement cache warming where we periodically update the cache and in the worst case deliver an outdated cache file while the new cache is built.

**Comment by freekmurze** [comment]
This currently isn't possible. The Laravel caching system doesn't support this use case.
